---
title: "Analysis Report 2: Your Title Here"
author: "Maggie Chen"
date: "November 21, 2018"
output: github_document
bibliography: references.bib
csl: bioinformatics.csl
---

*Overall, a single-spaced page is about 500 words. So if the guidelines say half of a page, think about writing around 250 words. You can use the wordcountaddin in RStudio to track your progress.*

# Introduction

Add about 1 page here. Must cite at least 5 peer reviewed articles.

# Methods

## Sample origin and sequencing

Add about half a page here. In this section instead of first person (I/we), use Li et al. [@li2015rna] and Seo et al. [@seo2012transcriptional], since you'll just be describing what they did, based on the methods in their paper(s). It's important to include this so the reader knows what the study design was for the data you will be presenting.

## Computational

These are the methods that were used to process the sequencing data. Should probably be at least a half of a page. At a very minimum should include citations for biomartr, trimmomatic, and sailfish. Note that these three methods references don't count towards the five references you need to cite in the introduction.

# Results

In addition to a minimum of 4-5 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion.

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("magrittr")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
```

```{r load-data, message = FALSE, echo = FALSE}
# load the dataset from a compressed binary file
# it gets loaded as an object called "joined_table"
# this has 6.2 million rows...so you will need to be thoughtful about
# how you analyze the data so that you don't overwhelm your laptop
load("output/final_compiled_counts/joined_count_data.RData")

# test that it loaded correctly before proceeding
stopifnot(exists("joined_table"))
```

```{r normal-never-table, echo = FALSE}
# Summary table of gene expression levels in the noraml tissue
# also, from never smokers
# The most highly expressed gene is listed in the talbe
top_15 <- joined_table %>%
  filter(normal_or_cancer == "normal") %>%
  filter(smoking_status == "never") %>%
  group_by(genename, cancer_stage) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 15)


# then use the `kable()` function to make a nicely formatted
# markdown table
top_15 %>%
  kable()
```

**Table 1**: 




```{r make-barplot-of-highly-expressed-genes, echo = FALSE}
# Use the higly expressed gene in
# never smoker's normal tissue found above 
# to generate a histogram of how those genes are expressed 
# in both normal and cancer tissue, along with different cancer stages 


# To pull out the unique gene names and turn
# them into a vector then use it below to make a histogram
# use the pull() funtion to get this as a vector
top_genes <- top_15 %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

# To generate a table use the mean count of gene expression
# also factors of cancer stages and tissue types

joined_table %>%
  filter(genename %in% top_genes) %>%
  filter(cancer_stage != "unknown") %>%
  # exclude samples has unknown stages
  group_by(normal_or_cancer, cancer_stage, genename) %>%
  summarise(mean_count = mean(counts_lengthscaledtpm)) %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = cancer_stage)) +
    geom_col(position = "dodge") +
  xlab("Genename") +
  ylab("Mean Expression Level") +
  ggtitle("Gene Expression Levels in Different Cancer Stages") +
  facet_grid(~ normal_or_cancer) +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 90,
                         hjust = 1))
```


**Figure 1**: The average gene expression levels in cancer and normal tissues. Categorized by smoking status, and colored by cancer stages. All four genes are highly expressed in the normal tissues, but they have different average expression levels in cancer tissues, and different cancer stages. EEF1A1 gene is relatively highly expressed in the cancer tissue. SFTPA1, SFTPA2, and SFTPC is down regulated in cancer tissues, in almost all cancer stages. SFTPC is mostly down regulated throughout all cancer stages. 

```{r barplot-of-SFTPC-expression-levels-smoking-cancer_stage}
# how SFTPC gene is expressed in all individuals(mean count of gene expression)
# samples from patients' both normal and cancer tissues
# along with the expression levels in different 
# cancer stages, and smoking status

joined_table %>%
  filter(genename %in% "SFTPC") %>%
  filter(smoking_status != "unknown") %>%
  filter(cancer_stage != "unknown") %>%
  group_by(normal_or_cancer, cancer_stage, smoking_status) %>%
  summarise(mean_count = mean(counts_lengthscaledtpm)) %>%
  ggplot(aes(x = smoking_status,
             y = mean_count,
             fill = cancer_stage)) +
    geom_col(position = "dodge") +
  xlab("Smoking Status") +
  ylab("Mean Expression Level") +
  ggtitle("SFTPC Expression Levels in Different Cancer and Smoking Status") +
  facet_grid(~ normal_or_cancer) +
   theme_bw() +
   theme(axis.text.x = element_text(angle = 90,
                         hjust = 1))
```

**Figure2**: In general, all cancer tissues has SFTPC gene significantly down regulated. Specifically, in never smokers, the SFTPC has showed some persistent to the down regulation in all cancer stages, especially later stages. For current and previous smokers, SFTPC are more down regulated than never smokers, with some outlier. For example, the 1B stage for the current smokers and the 3A stage for the previouslt smokers. 

```{r scatterplot-of-SFTPC-expression-smoking-age}
# Examine the average expression levels of SFTPC gene
# only for individals has both normal and cancer 
# tissue sampled, which based on smoking status
# and age at diagnosis

joined_table %>%
  filter(genename == "SFTPC") %>%
  filter(smoking_status != "unknown") %>%
  filter(normal_rnaseq == "yes") %>%
  group_by(normal_or_cancer, smoking_status, age_at_diagnosis) %>%
  summarise(mean_count = mean(counts_lengthscaledtpm)) %>%
  ggplot(aes(x = age_at_diagnosis,
             y = mean_count,
             color = smoking_status)) +
  xlab("Age at Diagnosis") +
  ylab("Mean Expression Level") +
  ggtitle("SFTPC Expression Levels Variants in Age and Smoking Status") +
  geom_point() +
   theme_bw() +
  geom_smooth(method = "lm",
              alpha = 0) +
  facet_grid(~ normal_or_cancer)

```

**Figure3**: Overall, SFTPC gene is highly down regulated in the cancer tissues. But highly expressed in normal tissues. Also, along with the smoking status of individuals. In cancer tissues, smoking will further decrease the gene expression levels along with increment of age at diagnosis. For previous and current smokers, SFTPC gene express in close to zero level and not change along with age change in the cancer tissue. In the cancer tissue from never smoker, the SFTPC expression level remains low, but increase with age increase. In normal tissues, for never smoker, the gene expression levels slightly decrease along with the increase of age. For normal tissues in current smokers, the gene expression is significantly decreased, while age increases. For normal tissue in previous smoker, the gene expression level increase, while age increases. 
 


```{r scaterplot-of-expression-level-of-gender-age-smoking}

# Due to Figure2, 3 has different trending of SFTPC gene expression
# in normal tissues, to investigate this different
# this plot shows how in different gender, and smoking status
# will influence the SFTPC genes expression levels over age

joined_table %>%
  filter(genename == "SFTPC") %>%
  filter(smoking_status != "unknown") %>%
  filter(normal_rnaseq == "yes") %>%
  group_by(normal_or_cancer, gender, smoking_status,
           cancer_stage, age_at_diagnosis) %>%
  summarise(mean_count = mean(counts_lengthscaledtpm)) %>%
  ggplot(aes(x = age_at_diagnosis,
             y = mean_count,
             color = smoking_status,
             shape = normal_or_cancer)) +
   xlab("Age at Diagnosis") +
  ylab("Mean Expression Level") +
  ggtitle("SFTPC Expression Level assosiate with Age, Gendner and Smoking") +
  geom_point() +
   theme_bw() +
   geom_smooth(method = "lm",
              alpha = 0) +
  facet_grid(~ gender)
  
```


**Figure4**: 





```{r make-boxplot-of-highly-expressed-genes, echo = FALSE}
# here we just want to pull out the unique gene names and turn
# them into a vector so we can use it below to make a boxplot
# we use the pull() funtion to get this as a vector, just like
# we did when making histograms several weeks ago
top_genes <- top_15 %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

# now we need to filter from the full data set again, because
# we don't just want summary data, we want all the data in
# order to make boxplots
joined_table %>%
  filter(genename == "SFTPC") %>%
  ggplot(aes(x = smoking_status,
             y = counts_lengthscaledtpm,
             fill = gender)) +
    geom_boxplot() +
    facet_grid(~ cancer_stage) +
    xlab("Gene Name") +
    ylab("Scaled read counts per gene") +
    ggtitle("Read counts per gene by gender and cancer stage") +
    theme_bw() + # simplifies theme
    theme(axis.text.x = # rotates x axis labels vertically
            element_text(angle = 90,
                         hjust = 1))
```


# Discussion

Add around 1-2 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
