---
title: "Smoking Further Downregulates the SFTPC Gene Expression in both Normal and Cancer Lung Tissues Increases the Risk of Lung Cancer"
author: "Maggie Chen"
date: "November 21, 2018"
output: github_document
bibliography: references.bib
csl: bioinformatics.csl
---

# Introduction
Lung cancer is the most deadly cancer for both men and women in the US, which had exceeded the death rate combination of three most common cancer that is the colon, breast and pancreatic cancer [@siegel2014cancer]. There are two main sub-type of lung cancer, small cell lung cancer and non-small cell lung cancer (NSCLC). NSCLC accounts for 85% of all lung cancers. NSCLC further classifies into three types, squamous-cell carcinoma, adenocarcinoma and large-cell carcinoma [@molina2008non]. Adenocarcinoma arises from the typeII alveolar cell that forms the small airways. Adenocarcinoma is also the most common lung cancer found in both smoking and non-smoking population which accounts 30-40% of lung cancer regardless of gender and age [@couraud2012lung]. Smoking is the most prevalent risk factor for lung cancer in both male and female population. The length of smoking time and the amount of cigarettes people smoked is positively correlated to the risk of lung cancer [@powell2013association]. Further study has also indicated the differences between smokers and nonsmoker's pathogenic mechanisms in lung cancer. For example, in adenocarcinoma, there is a distinct new molecular pathway in smoker's lung cancer development than non-smokers. What's more, those smoking-related genomic signatures in NSCLC can help predict cancer with 74% of accuracy [@massion2008smoking].

Nowadays, along with the improvement of sequencing techniques, RNA-Seq has become a sufficient method use the deep sequencing technology to profile transcriptome. RNA-Seq has provided a new view of eukaryotic transcriptome complexity. RNA-Seq also enables precise measurement of levels of transcripts and isoforms [@wang2009rna]. Compare to other sequencing technologies, like microarray, RNA-Seq has less background noise, and it is more sensitive regarding low abundance and high-fold expression change detection [@zhao2014comparison]. As one of the examples, RNA-Seq was used to provide an insight of lung cancer development and its correlation with smoking. Its found that there are 27 genes expressed differently in human airway epithelial cells between smoker that has lung cancer and non-smoker without lung cancer [@cheng2012comparison]. Li et al. has found several genes are either down-regulated or up-regulated in lung adenocarcinoma tissues using the data set from Seo et al., in which both Li et al. and Seo et al. have found smoking will contribute to the induction of gene expression dysfunction in tumor tissue, especially down-regulation of gene expression in tumor tissues [@li2015rna]. Then I was primarily interested in the questions of how lung  adenocarcinoma will impact the lung function and the expression levels of the genes that support lung functions, in addition to genes identified by Li et al. 

Thus, the goal for this paper is to investigate the impact of smoking on genes that are highly expressed in healthy tissue, but down-regulated in cancer tissues. Also, to investigate whether there is a correlation between smoking, age, and gender on the gene expression level. I have found the SFTPC gene is highly downregulated in cancer tissues, which encodes the pulmonary-associated surfactant protein C (SPC). SPC is exclusively produced by type II alveolar cells, which prevents lung collapse by lowering the surface tension at the air-liquid interface in the lung [@clark2005genetics]. What's more, smoking cause the further down-regulation of SFTPC gene along with the increase of age at diagnosis. However, for previous and never smoker, there is no significant correlation between age and SFPTC gene expression levels. By understanding the underlying driven mutation or mechanism of lung adenocarcinoma, like when SFTPC gene is extensively downregulated by smoking, to increase the survival rate of patients, we can provide support to maintain the surface tension of alveolar cells. This will help provide a possible treatment to lung adenocarcinomas, especially when the patients is a current smoker. Understanding the underlying driver mutations that are induced by smoking can help us narrow down a range of possible mutation that threats the patients and provides adequate treatment faster. 

# Methods

## Sample origin and sequencing

Seo et al. sampled the original data from patients for both cancer and adjacent paired-normal tissue specimens via surgical specimens. Both cancer and normal tissues specimens were grossly dissected and preserved in liquid nitrogen immediately after surgery. Seo et al. used RNAiso Plus (Takara Bio Inc.) to extracted RNA from tissues for RNA-Seq, which is followed by using RNeasy MinElute (Qiagen Inc.) to purify the RNA samples. Assessment of RNA quality was quantified by using RNA 6000 Nano LabChip on a 2100 Bioanalyzer (Agilent Inc.). Seo et al. extracted genomic DNA from normal lung tissues for exome sequencing. Genomic DNA from each sample was sheared and used for the construction of paired-end sequence library from Illumina. The SureSelect Human all Exon 50Mb Kit (Agilent Inc.) was used according to the manufacturer's instruction to enrich the exonic sequences. Illumina TruSeq SBS Kit v3 on a HiSeq 2000 sequencer (Illumina Inc.) was used to sequence libraries for RNA and exome sequencing, 100-bp paired-end reads were obtained. Seo et al. also used the Illumina pipeline to perfom the image analysis and base calling with default settings. Total 200 of specimen samples was taken from patients, and 87 was sequenced using RNA-Seq. For those 87 individuals whose cancer specimens was RNA sequenced by Seo et al., smoking history before the diagnosis was provided for 83 of them, which includes 47 smokers, 36 never-smokers, and four unknowns. 23 out of 47 smokers has the amount of smoking available in the unit of pack-years [@seo2012transcriptional].
Li et al. used the sequence data from Seo et al., Li et al. used the 68 lung adenocarcinoma patients' transcriptome sequences with validated smoking status from accession number GSE40419. Overall, there were about 14 billion paired-end sequence read an average length in 101bp. Also the phenotype data from the public website (http://genome.cshlp.org/content/22/11/2109/suppl/DC1), which include gender, age at diagnosis, and smoking history. There are 34 smokers and 34 non-smokers in the 68 total samples. Li et al. have also downloaded RNA-Seq data from six non-smoker patients from accession number GSE337765.

## Computational

First of all, I created a bash script to download the RNA-Seq sequencing file and install tools for analysis. Then I created an R script to install biomartr package which helps retrieve all or selected genomic data, promote, coding sequences, and annotations in NCBI [@drost2017biomartr]. Thus, I used it to download the reference genome, transcript, and genome annotations from NCBI human genome RefSeq. Then I converted the paired-end sequence sra files into fastq files using a bash script. After that, I created index of kmers for sailfish quasi-aligner in a bash script. Sailfish is a novel computational method that avoids mapping read to quantify the abundance of previously annotated RNA isoform from RNA-Seq data at a fast speed [@patro2014sailfish]. After it, I use the bash script to run the fastqc on all fastq files and save the output to the output directory. Then I trimmed the paired-end reads in parallel using trimmonatic which is a tool to trim Illumina NGS data, help remove the adopter, remove low-quality bases [@bolger2014trimmomatic]. In this project, I trimmed any reads that drop below 20 in a window size of 4 based on the fastqc output. To ensure all the sequence files are trimmed I created another script to check whether the files are trimmed, if not then run the trimmomatic. Then use the result of trim, I make a script to run sailfish to count all reads from trimmed reads where both paired reads (Forward and Reverse) made it through trimmomatic check [@patro2014sailfish]. Then, in order to construct a transcript to the gene ID mapping table. In the bash script, I firstly construct a non-redundant list of Genbank ID to gene names, which is from sailfish quant output and make that the first column of output file. Secondly, grep over first column in quant.sf files to find Genbank ID and then add gene name as the second column, and make sure this gets loaded into R. Lastly, I created an R script to read in all of the transcript counts made by sailfish individually on each sample, with aggregate the transcript level counts to the gene level. Then normalize them based on length and read depth, and then build a table with rows as genes and columns as samples, with each cell being the normalized count for that gene in that sample. Then, I joined the two other metadata files we have, from SRA and from the supplementary table from the original manuscript that has information like patient age and smoking behavior. I also melted the output file only to include interesting columns, and write out the output in compressed csv format as well as binary RData format.


# Results

First of all, to investigate the critical genes that maintain the healthy lung function. I listed out the top fifteen most highly expressed genes in never smokers' normal tissues (Table1). The expression levels are measured as the mean count of normalized expression count and then categorized in cancer stages. For never smokers that has stage 3A lung cancer, in their normal tissue the SFTPC gene has the highest average expression level across all samples, then SFTPA2 gene in stage 3A never smokers, and then the EEF1A1 gene in stage 3B never smokers is also relatively highly expressed. Each sample is recognized as an independent individual and grouped into cancer stages which creates some repeated genes names (Table1). What's more, most of cancer stages that has high gene expression levels are in stage 3, which is the relative later and sever stages. So, I extracted the unique genes names from the top 15 highly expressed gene list, that is SFTPC, SFTPA1, SFTPA2, EEF1A1, and SFTPB gene, then I examined their expression levels in different tissue type (normal or cancer), and different cancer stages. As a result, all five genes are highly expressed in the normal tissues, but they have different average expression levels in cancer tissues, and different cancer stages. EEF1A1 gene is relatively highly expressed in the cancer tissue with minimal down-regulation, mostly on stage 3B and 4. SFTPA1, SFTPA2, and SFTPC are more down-regulated in cancer tissues then normal tissues, in almost all cancer stages. SFTPA1 and SFTPA2 gene expression are relatively down-regulated in stages other 2B. SFTPB gene is mostly down-regulated in stage 3B and stage 4. SFTPC is mostly down-regulated throughout all cancer stages. As SFTPC gene is mostly down-regulated in cancer tissues, more importantly throughout all cancer stages (Figure1). 

Next, I specifically focused on how the SFTPC gene expression level is impacted by smoking. Overall, all cancer tissues have SFTPC gene significantly down-regulated. Specifically, for current and previous smokers' cancer tissues, most of the SFTPC gene expression is more downregulated than never smokers. But there are several outliers, for example, the 1B stage for the current smokers and the 3A stage for the previously smokers (Figure2). In contrast, in never smokers' cancer tissues, the SFTPC has shown some persistent to the down-regulation in all cancer stages, especially later stages. Also, in the normal tissues, current smokers have a lower SFTPC gene average expression levels than never and previous smokers (Figure2). After observing current smoking status will impact the SFTPC gene expression levels with some irregular pattern, then I further include the age at diagnosis as a variable to investigate how smoking will impact the SFTPC gene expression along with age at diagnosis in both normal and cancer tissues. SFTPC gene is highly down-regulated in the cancer tissues, which has average gene expression levels close to zero. For previous and current smokers' cancer tissues, SFTPC gene expression is closer to zero than never smokers and not change along with age change in the cancer tissue. In the cancer tissue from never smokers, the SFTPC expression level remains low, but there is a slight increase with age increase due to two outliers at the age of 65; otherwise, there is a small correlation with age at diagnosis. In contrast, the SFTPC gene is significantly highly expressed in normal tissues. Also, along with the smoking status of individuals, in normal tissues, for never smoker, the gene expression levels slightly decrease along with the increase of age, which is insignificant. For normal tissues from current smokers, the gene expression level decreased significantly, while age increases. For normal tissue from previous smokers, the gene expression level increase is not significantly, while age increases (Figure3). Knowing current smoking status will decrease SFTPC expression in normal tissues along with age at diagnosis, I was curious about how gender will associate with it. 

Thus, lastly, I examined the relationship between the average SFTPC gene expression levels, gender and smoking status. As a result, comparing the SFTPC gene expression levels in male and female independently, cancer tissues has SFTPC gene significantly down-regulated than normal tissues. Compare across male and female cancer tissue samples; female has SFTPC expressed slightly more than males cancer tissues. In the comparison of tissue types, female samples are dominated by never smokers with a few previous smokers' samples. Male samples are abundant in previous smokers and current smokers, with a minimal sample amount of never smokers. Thus, for female never smokers the SFTPC expression levels are not changed significantly along with age at diagnosis. For the current male smoker, SFTPC gene's mean expression levels decrease significantly while the age at diagnosis increases. For previous male smokers, age at diagnosis does not influence the SFTPC expression levels (Figure4). 
 

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("magrittr")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
```

```{r load-data, message = FALSE, echo = FALSE}
# load the dataset from a compressed binary file
# it gets loaded as an object called "joined_table"
# this has 6.2 million rows...so you will need to be thoughtful about
# how you analyze the data so that you don't overwhelm your laptop
load("output/final_compiled_counts/joined_count_data.RData")

# test that it loaded correctly before proceeding
stopifnot(exists("joined_table"))
```

```{r normal-never-table, echo = FALSE}
# Summary table of gene expression levels in the noraml tissue
# also, from never smokers
# The most highly expressed gene is listed in the talbe
top_15 <- joined_table %>%
  filter(normal_or_cancer == "normal") %>%
  filter(smoking_status == "never") %>%
  group_by(genename, cancer_stage) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 15)


# then use the `kable()` function to make a nicely formatted
# markdown table
top_15 %>%
  kable()
```

**Table 1**: Top fifteen genes that is most highly expressed in never smoker's normal tissues.
The expression levels are measured as the mean count of normalized expression count and then categorized in cancer stages. For never smokers that has stage 3A lung cancer, in their normal tissue the SFTPC gene has the highest average expression level across all samples, then SFTPA2 gene in stage 3A never smokers, and then the EEF1A1 gene in stage 3B never smokers is also relatively highly expressed. Each sample is recognized as an independent individual and grouped into cancer stages which creates some repeated genes in the table.



```{r make-barplot-of-highly-expressed-genes, echo = FALSE}
# Use the higly expressed gene in
# never smoker's normal tissue found above 
# to generate a histogram of how those genes are expressed 
# in both normal and cancer tissue, along with different cancer stages 


# To pull out the unique gene names and turn
# them into a vector then use it below to make a histogram
# use the pull() funtion to get this as a vector
top_genes <- top_15 %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

# To generate a table use the mean count of gene expression
# also factors of cancer stages and tissue types

joined_table %>%
  filter(genename %in% top_genes) %>%
  filter(cancer_stage != "unknown") %>%
  # exclude samples has unknown stages
  group_by(normal_or_cancer, cancer_stage, genename) %>%
  summarise(mean_count = mean(counts_lengthscaledtpm)) %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = cancer_stage)) +
    geom_col(position = "dodge") +
  xlab("Genename") +
  ylab("Mean Expression Level") +
  ggtitle("Gene Expression Levels in Different Cancer Stages") +
  facet_grid(~ normal_or_cancer) +
  theme_bw() +
  theme(axis.text.x = element_text(angle = 90,
                         hjust = 1))
```


**Figure 1**: The average gene expression levels in cancer and normal tissues. Categorized by smoking status, and colored by cancer stages. All five genes are highly expressed in the normal tissues, but they have different average expression levels in cancer tissues, and different cancer stages. EEF1A1 gene is relatively highly expressed in the cancer tissue with minimal down-regulation, mostly on stage 3B and 4. SFTPA1, SFTPA2, and SFTPC are more down-regulated in cancer tissues then normal tissues, in almost all cancer stages. SFTPA1 and SFTPA2 gene expression are relatively down-regulated in stages other 2B. SFTPB gene is mostly down-regulated in stage 3B and stage 4. SFTPC is mostly down-regulated throughout all cancer stages. As SFTPC gene is mostly down-regulated in cancer tissues, more importantly throughout all cancer stages 

```{r barplot-of-SFTPC-expression-levels-smoking-cancer_stage, echo = FALSE}
# how SFTPC gene is expressed in all individuals(mean count of gene expression)
# samples from patients' both normal and cancer tissues
# along with the expression levels in different 
# cancer stages, and smoking status

joined_table %>%
  filter(genename %in% "SFTPC") %>%
  filter(smoking_status != "unknown") %>%
  filter(cancer_stage != "unknown") %>%
  group_by(normal_or_cancer, cancer_stage, smoking_status) %>%
  summarise(mean_count = mean(counts_lengthscaledtpm)) %>%
  ggplot(aes(x = smoking_status,
             y = mean_count,
             fill = cancer_stage)) +
    geom_col(position = "dodge") +
  xlab("Smoking Status") +
  ylab("Mean Expression Level") +
  ggtitle("SFTPC Expression Levels in Different Cancer and Smoking Status") +
  facet_grid(~ normal_or_cancer) +
   theme_bw() +
   theme(axis.text.x = element_text(angle = 90,
                         hjust = 1))
```

**Figure2**: SFTPC gene expression levels' association with smoking status and different cancer stages, tissue types. Categorized by smoking status and tissue types, colored by cancer stages. Overall, all cancer tissues has SFTPC gene significantly down regulated. Specifically, for current and previous smokers' cancer tissues there is a irregular, but mostly more down regulation than never smokers with some out-liers. For example, the 1B stage for the current smokers and the 3A stage for the previously smokers. In contrast, in never smokers' cancer tissues, the SFTPC has showed some persistent to the down regulation in all cancer stages, especially later stages. In addition, in the normal tissues, current smoker has a lower SFTPC gene average expression levels than never and previous smokers. 

```{r scatterplot-of-SFTPC-expression-smoking-age, echo = FALSE}
# Examine the average expression levels of SFTPC gene
# only for individals has both normal and cancer 
# tissue sampled, which based on smoking status
# and age at diagnosis

joined_table %>%
  filter(genename == "SFTPC") %>%
  filter(smoking_status != "unknown") %>%
  filter(normal_rnaseq == "yes") %>%
  group_by(normal_or_cancer, smoking_status, age_at_diagnosis) %>%
  summarise(mean_count = mean(counts_lengthscaledtpm)) %>%
  ggplot(aes(x = age_at_diagnosis,
             y = mean_count,
             color = smoking_status)) +
  xlab("Age at Diagnosis") +
  ylab("Mean Expression Level") +
  ggtitle("SFTPC Expression Levels Variants in Age and Smoking Status") +
  geom_point() +
   theme_bw() +
  geom_smooth(method = "lm",
              alpha = 0) +
  facet_grid(~ normal_or_cancer)

```

**Figure3**: SFTPC gene expression levels' association with age at diagnosis and smoking status. Use age at diagnosis as the continuous variable and categorize into tissues types and colored by smoking status.  SFTPC gene is highly down-regulated in the cancer tissues, which has average gene expression levels close to zero. For previous and current smokers' cancer tissues, SFTPC gene expression is closer to zero than never smokers and not change along with age change in the cancer tissue. In the cancer tissue from never smokers, the SFTPC expression level remains low, but there is a slight increase with age increase due to two outliers at the age of 65; otherwise, there is a small correlation with age at diagnosis. In contrast, the SFTPC gene is significantly highly expressed in normal tissues. Also, along with the smoking status of individuals, in normal tissues, for never smoker, the gene expression levels slightly decrease along with the increase of age, which is insignificant. For normal tissues from current smokers, the gene expression decreased significantly, while age increases. For normal tissue from previous smokers, the gene expression level increase not significantly, while age increases.

```{r scaterplot-of-expression-level-of-gender-age-smoking, echo = FALSE}

# Due to Figure2, 3 has different trending of SFTPC gene expression
# in normal tissues, to investigate this different
# this plot shows how in different gender, and smoking status
# will influence the SFTPC genes expression levels over age

joined_table %>%
  filter(genename == "SFTPC") %>%
  filter(smoking_status != "unknown") %>%
  filter(normal_rnaseq == "yes") %>%
  group_by(normal_or_cancer, gender, smoking_status,
           cancer_stage, age_at_diagnosis) %>%
  summarise(mean_count = mean(counts_lengthscaledtpm)) %>%
  ggplot(aes(x = age_at_diagnosis,
             y = mean_count,
             color = smoking_status,
             shape = normal_or_cancer)) +
   xlab("Age at Diagnosis") +
  ylab("Mean Expression Level") +
  ggtitle("SFTPC Expression Level assosiate with Age, Gender and Smoking") +
  geom_point() +
   theme_bw() +
   geom_smooth(method = "lm",
              alpha = 0) +
  facet_grid(~ gender)
```


**Figure4**: SFTPC gene expression levels in different genders and its correlation with age at diagnosis and smoking status. Age at diagnosis is the continuous variable and categorized into gender and colored by smoking status, points are shaped by tissue types. Compare across male and female cancer tissue samples; female has SFTPC expressed slightly more than males cancer tissues. In the comparison of tissue types, female samples are dominated by never smokers with a few previous smokers' samples. Male samples are abundant in previous smokers and current smokers, with a minimal sample amount of never smokers. Thus, for female never smokers the SFTPC expression levels are not changed significantly along with age at diagnosis. For the current male smoker, SFTPC gene's mean expression levels decrease significantly while the age at diagnosis increases. For previous male smokers, age at diagnosis does not influence the SFTPC expression levels.




# Discussion
The five highly expressed genes in never smokers' normal tissues are SFTPC, SFTPA1, SFTPA2, EEF1A1, and SFTPB gene. They all serve different roles in lung functions. EEF1A1 encodes for a subunit of Eukaryotic translation elongation factor-1, which help delivery aminoacyl tRNA to the ribosome [@sanders1996immunofluorescence]. This is very likely to be a housekeeping gene that even cancer cells need translation to produce proteins; thus it is not significantly down-regualted in cancer tissues when comparing to normal tissues. SFTPA1 and SFTPA2 both belong to the collection of SPA, which helps noninflammatory in the lung by stimulating immunoreceptor on alveolar cells [@greene2003binding]. Cancer development has involved tightly with inflammatory responses of immunosystem but not much about the noninflammatory responses [@coussens2002inflammation]. Thus SFTPA1 and SFTPA2 are not highly down-regulated in cancer tissues. SFTPB gene encodes for pulmonary-associated surfactant B protein (SPB), which has a similar function as SFTPC that is to help prevent lung collapse and lowering surface tension at the air-liquid interface of alveoli in the lung. Both SFTPC and SFTPC encodes proteins as part of the surfactant [@clark1995targeted]. SFTPB gene has also been down-regulated in cancer tissues, but not as much as SFPTPC gene (Figure1). However, lung adenocarcinoma specifically target the SFTPC might due to SFTPC is exclusively expressed on type II alveolar cells in the lung than SFTPB gene, to reach higher efficiency. However, more investigation is needed. 

In conclusion, in the same individual lung cancer tissue has a significant lower SFTPC expression level, which might lead to a significant loose of lung function due to the lung collapses. SFTPC is an essential gene for typeII alveolar cells in the lung, which is also the main location of adenocarcinoma development. Morever, this loose of normal lung function will lead to a high death rate of lung adenocarcinoma since the SFTPC gene is vital for breathing and oxygen transportation from air to blood. Besides, typeII alveolar cells also protect lung from invasion of pathogens, which means the loose of alveolar typeII cell function will increase the chance of getting lung infections [@fehrenbach2001alveolar]. Also, smoking, especially current male smokers will have more down-regulated SFTPC gene expression levels in normal tissues along with the increase of age at diagnosis. This provides one of the possible reasoning that adenocarcinoma is the most common type of NSCLC, due to the large smoking population. The decreased SFTPC expression in typeII alveolar will increase the chance of smoker patients affected by adenocarcinoma NSCLC. Also, the elderly male current smoking population is more vulnerable to the adenocarcinoma NSCLC. However, previous male smokers are less likely to experience the extensive down-regulation of SFTPC expression levels as the current male smokers. What's more, for female never-smokers, SFTPC gene expression levels remain relatively constant along with the increase of age at diagnosis. Thus, stop smoking as soon as possible or stay away from smoking will be sufficient to increase the survival rate of lung cancer, specifically adenocarcinoma NSCLC. 

There are also several limitations to this paper. Firstly, the female samples are dominated by never smokers; thus we need more data and samples to ensure the observation and conclusions reached in male samples can apply to female populations. Also, for male samples there is a minimal amount of never smokers. Thus, we need more samples to ensure that the conclusion we reached in females can apply to male never smokers. Secondly, SFTPC is highly expressed in type II alveolar cells [@clark2005genetics]. However, all the samples are from lung adenocarcinoma patients; thus I can only limit the finding of how SFTPC is impacted by smoking to lung adenocarcinoma. Also, all the samples are from patients in Korean, which is not inclusive to all the human population. Thus, we need to include more diverse human  samples to investigate whether the down-regulation of SFTPC gene by smoking applies to all human populations. Lastly, for previous smokers, there is no data about how many years they have quit smoking. There might be a correlation between the time of stop smoking and re-store of SFTPC expression levels. Thus, have a more variant sample set to include different types of cancer, human race and years of stop smoking will further expand the applications of this finding or provide a race-specific effect of smoking on lung cancers. 

In summary, the SFTPC gene is highly expressed in more lung typeII alveolar cells, which is significant to lung function. Lung cancer will significantly down-regulate the expression of the SFTPC gene in all cancer stages. Furthermore, smoking, especially current smoking status will make patients more vulnerable to lung cancer, and at older ages of diagnosis. In order to further down-regulate the SFTPC gene expression, there might be some new pathway activated by the compound in cigarettes to down-regulate the SFTPC gene expression directly or up-regulate the regulatory factor of SFTPC gene expression. However, this still remains in question and needs further research and evidence. What's more, based on the understanding of smoking down-regulatea the SFTPC expression levels in normal tissue, this can function as a prediction to the likely-hood of the individual to get lung adenocarcinoma or even other lung diseases. Also, for current smokers, lung adenocarcinoma treatment that helps restore the surface tension of lung cells can help improve the survival rate.


# Sources Cited
